spec/controllers/users/rules_of_use_controller_spec.rb

```diff
@@ -1,6 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe Users::RulesOfUseController do
+  let(:rules_of_use_updated_at) { 1.day.ago }
+  let(:accepted_terms_at) { nil }
+  let(:user) { create(:user, :signed_up) }
+  before do
+    user.accepted_terms_at = accepted_terms_at
```

Comment:
Taking a closer look at this, think we should patch the factory to remove this from `after_build` and `after_stub`, and just make this a normal attribute, so we don't need this extra setter step.

I confirmed this worked on the current `main` and seems to let this file's specs pass

```diff
--- a/spec/factories/users.rb
+++ b/spec/factories/users.rb
@@ -8,9 +8,10 @@ FactoryBot.define do
       with { {} }
       email { Faker::Internet.safe_email }
       confirmed_at { Time.zone.now }
-      accepted_terms_at { Time.zone.now }
     end
 
+    accepted_terms_at { Time.zone.now if email }
+
     after(:build) do |user, evaluator|
       next unless user.email_addresses.empty?
       user.email_addresses.build(
@@ -19,7 +20,6 @@ FactoryBot.define do
       )
       user.email = evaluator.email
       user.confirmed_at = evaluator.confirmed_at
-      user.accepted_terms_at = Time.zone.now
     end
 
     after(:stub) do |user, evaluator|
@@ -30,7 +30,6 @@ FactoryBot.define do
       )
       user.email = evaluator.email
       user.confirmed_at = evaluator.confirmed_at
-      user.accepted_terms_at = Time.zone.now
     end
```